codegeex.el

```diff
@@ -3,8 +3,9 @@
 ;; Copyright (C) 2023 Hao Zhang
 
 ;; Author: Hao Zhang <hzhangxyz@outlook.com>
+;; Contributor: Samuel Dawant <samueld@mailo.com>
 ;; Keywords: codegeex, completion
-;; Version: 0.0.1
+;; Version: 0.1.0
```

Comment:
I took the liberty to increase the version (medium one since it add a functionality) and put myself in the contributor ;). If I may